docs/pywbemclifeatures.rst

```diff
@@ -289,20 +289,44 @@ The following are examples of scalar property definitions:
 Interactively selecting INSTANCENAME
 ------------------------------------
 
-The INSTANCENAME argument has a certain complexity, particularly for
-associations and for classes with multiple keys.
+The INSTANCENAME argument can be complex, particularly for associations and
+classes with multiple keys.
+
+To simplify using INSTANCENAME on the command line, pywbemcli provides a wild
+card character "?" that can be used with commands in place of the INSTANCENAME
+keys on commands that have an INSTANCENAME argument. This allows the user to
+specify only the class name (as the INSTANCENAME argument value) with the wild
+card for the keys. Pywbemcli retrieves all instance names of that class from
```

Comment:
The sentence "This allows the user ..." basically says what was introduced in the previous sentence can actually be used. I assume you mean to convey that the syntax is class name followed by the wild card for keys, but to me that is not what it says, plus it is not correct because the dot would be missing in that interpretation.
I suggest replacing this sentence with an ABNF syntax definition.

Reply:
Redid to add ABNF